dev/source/docs/building-setup-windows-cygwin.rst

```diff
@@ -129,7 +129,8 @@ Set up directories/paths and extra packages in Cygwin
 
     ln -s /usr/bin/python3.7 /usr/bin/python
     ln -s /usr/bin/pip3.7 /usr/bin/pip
-    pip install empy pyserial pymavlink
+    pip install pyserial pymavlink
+    pip install empy==3.3.4
```

Comment:
```suggestion
    pip install pyserial pymavlink empy<4.0
```
Doing it all in one command will let pip resolve any dependency conflicts. Let's also allow for any possible updates to the empy 3 line. I doubt there will be any, but let's leave the door open.